codecov.yml

```diff
@@ -18,12 +18,24 @@ component_management:
         branches:
           - '!main'
   individual_components:
-    - component_id: api-graphql # this is an identifier that should not be changed
-      name: Portal GraphQL
+    - component_id: api-graphql
+      name: Portal GraphQL (Not Prioritised)
       paths:
         - apps/api/**
         - libs/api/domains/**
-    - component_id: application-system-api-graphql # this is an identifier that should not be changed
+    - component_id: microservices
+      name: All REST APIs except Application System API.
+      paths:
+        - apps/services/**
+        - apps/air-discount-scheme/backend/**
+        - apps/download-service/**
+        - libs/auth-api-lib/**
```

Comment:
We talked about when rethinking the folder structure of our repo that if service requires a library it should have `libs/services/<app name>` to we want to list that path as well? 
Today it is only us using it for shared testing utilities for auth services.

Reply:
Good point